[amd-amf] Build error on x64-linux

Package: amd-amf:x64-linux@1.4.36
  
  **Host Environment**
  
  - Host: x64-linux
  - Compiler: GNU 13.3.0
  -    vcpkg-tool version: 2025-04-16-f9b6c6917b23c1ccf16c1a9f015ebabf8f615045
      vcpkg-scripts version: e9d0d6fe71 2025-04-29 (10 hours ago)
  
  **To Reproduce**
  
  `vcpkg install `
  
  **Failure logs**
  
  ```
  Downloading https://github.com/GPUOpen-LibrariesAndSDKs/AMF/archive/v1.4.36.tar.gz -> GPUOpen-LibrariesAndSDKs-AMF-v1.4.36.tar.gz
  GPUOpen-LibrariesAndSDKs-AMF-v1.4.36.tar.gz.112994.part: error: download from https://github.com/GPUOpen-LibrariesAndSDKs/AMF/archive/v1.4.36.tar.gz had an unexpected hash
  note: Expected: 589fccabaadb27e48e9adb1d3594db2adadee343c966f8db99ff29a92ec78ae6b0c42f13113a4fc66da0044ee660cfa1caf6867c508af044935646c09f5af50e
  note: Actual  : e19f8f98448412812ea1a4bf677ea501bebfc37871160e1cd0d0d2bf91af22f2[1154](https://github.com/RealChuan/Vcpkg-Examples/actions/runs/14749116214/job/41402227698#step:9:1155)06949b594f405dab153952dcc3cbdc666ef2e6be1b768b803cdde7e23a7b
  CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:136 (message):
    Download failed, halting portfile.
  Call Stack (most recent call first):
    scripts/cmake/vcpkg_from_github.cmake:120 (vcpkg_download_distfile)
    buildtrees/versioning_/versions/amd-amf/bd224304fd2caeb6f476511884069744e4b88f8f/portfile.cmake:1 (vcpkg_from_github)
    scripts/ports.cmake:206 (include)
  
  
  
  ```
  
  **Additional context**
  
  <details><summary>vcpkg.json</summary>
  
  ```
  {
    "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
    "name": "manifest",
    "version": "0.0.1",
    "description": "manifest mode test",
    "dependencies": [
      "benchmark",
      "breakpad",
      "crashpad",
      "efsw",
      "giflib",
      "glfw3",
      "glm",
      "glog",
      "gtest",
      "protobuf",
      "stb",
      "vulkan-hpp",
      {
        "name": "curl",
        "features": [
          "openssl",
          "brotli",
          "c-ares",
          "http2",
          "tool"
        ]
      },
      {
        "name": "ffmpeg",
        "features": [
          "ass",
          "bzip2",
          "ffmpeg",
          "ffplay",
          "ffprobe",
          "freetype",
          "fribidi",
          "gpl",
          "opengl",
          "x264",
          "x265",
          "zlib",
          {
            "name": "amf",
            "platform": "windows | linux"
          },
          {
            "name": "nvcodec",
            "platform": "windows | linux"
          },
          {
            "name": "qsv",
            "platform": "windows"
          }
        ]
      },
      {
        "name": "glad",
        "features": [
          "extensions"
        ]
      },
      {
        "name": "openssl",
        "features": [
          "tools"
        ]
      }
    ],
    "overrides": [
      {
        "name": "openssl",
        "version-string": "3.5.0"
      }
    ],
    "builtin-baseline": "e9d0d6fe71ec440a6a7906f92f941a181b937556"
  }
  
  ```
  </details>

--- a/ports/amd-amf/portfile.cmake
+++ b/ports/amd-amf/portfile.cmake
@@ -2,7 +2,7 @@ vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO GPUOpen-LibrariesAndSDKs/AMF
     REF "v${VERSION}"
-    SHA512 589fccabaadb27e48e9adb1d3594db2adadee343c966f8db99ff29a92ec78ae6b0c42f13113a4fc66da0044ee660cfa1caf6867c508af044935646c09f5af50e
+    SHA512 e19f8f98448412812ea1a4bf677ea501bebfc37871160e1cd0d0d2bf91af22f2115406949b594f405dab153952dcc3cbdc666ef2e6be1b768b803cdde7e23a7b
     HEAD_REF master
 )
 

--- a/ports/amd-amf/vcpkg.json
+++ b/ports/amd-amf/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "amd-amf",
   "version": "1.4.36",
+  "port-version": 1,
   "description": "AMD Advanced Media Framework headers",
   "homepage": "https://github.com/GPUOpen-LibrariesAndSDKs/AMF",
   "license": "MIT",

--- a/versions/a-/amd-amf.json
+++ b/versions/a-/amd-amf.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "f2de56dcf603903f753764e5b3cfd413b58ed323",
+      "version": "1.4.36",
+      "port-version": 1
+    },
+    {
       "git-tree": "bd224304fd2caeb6f476511884069744e4b88f8f",
       "version": "1.4.36",
       "port-version": 0

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -110,7 +110,7 @@
     },
     "amd-amf": {
       "baseline": "1.4.36",
-      "port-version": 0
+      "port-version": 1
     },
     "ampl-asl": {
       "baseline": "1.0.1",

